helm/idol-answerserver/Chart.yaml

```
@@ -0,0 +1,37 @@
+# BEGIN COPYRIGHT NOTICE
+# Copyright 2024 Open Text.
+# 
+# The only warranties for products and services of Open Text and its affiliates and licensors
+# ("Open Text") are as may be set forth in the express warranty statements accompanying such
+# products and services. Nothing herein should be construed as constituting an additional warranty.
+# Open Text shall not be liable for technical or editorial errors or omissions contained herein.
+# The information contained herein is subject to change without notice.
+#
+# END COPYRIGHT NOTICE
+apiVersion: v2
+name: idol-answerserver
+version: "0.1.0"
+appVersion: "24.1"
+description: Adds IDOL ANSWERSERVER
+dependencies:
+- name: idol-library
+  version: "0.4.0"
+  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
+- name: single-content
+  alias: "passageextractorSingleAgentstore"
+  condition: passageextractorSingleAgentstore.enabled
+  version: "0.4.0"
+  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
+- name: single-content
+  alias: "answerbankSingleAgentstore"
+  condition: answerbankSingleAgentstore.enabled
```

```suggestion
  alias: "answerbankAgentstore"
  condition: answerbankAgentstore.enabled
```
Similarly don't need single in here (and related .Values changes)